android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/MainActivity.java

```diff
@@ -452,6 +453,11 @@ public boolean onOptionsItemSelected(MenuItem item) {
         KMManager.getUpdateTool().executeOpenUpdates();
         // Dismiss icon
         updateUpdateCountIndicator(0);
+        final MenuItem _keyboardupdate = menu.findItem(R.id.action_update_keyboards);
```

Comment:
Is it possible for `menu` to be uninitialized at this point?

Reply:
The system should have called onCreateOptionsMenu() the first time menu is displayed

https://developer.android.com/reference/android/app/Activity#onCreateOptionsMenu(android.view.Menu)